en/docs/learn/tutorials/expose-a-soap-service-as-a-rest-api.md

```diff
@@ -59,7 +59,17 @@ This tutorial will explain the steps to design, publish and invoke a SOAP servic
      see [Enforce Throttling and Resource Access Policies]({{base_path}}/learn/rate-limiting/setting-throttling-limits/).</p>
      </div>
      </html>
-     
+
+    <html><div class="admonition note">
+     <p class="admonition-title">Note</p>
```

Comment:
We need to do the [these changes](https://github.com/wso2/docs-apim/pull/2761#pullrequestreview-557258172) in this(3.2.0) docs as well